### PR TITLE
Nightly : 1.7.8.x - Move to Github Actions

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Nightly : 1.7.8.x - Move to Github Actions
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: & Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/5257106124
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

File: .docker/Dockerfile [hunk 1]
```diff
@@ -3,9 +3,12 @@ ARG VERSION
 
 FROM prestashop/prestashop-git:$VERSION
```
Comment (boubkerbribri) [inline .docker/Dockerfile:4]:
```suggestion
FROM prestashop/base:$VERSION
```

Comment (boubkerbribri) [reply .docker/Dockerfile:4]: to make it work on ga.ui.tests.pr

Comment (Progi1984) [reply .docker/Dockerfile:4]: No need. That works.

**Review by boubkerbribri — CHANGES_REQUESTED** [review]
Need to remove files on UI tests folder :
- .docker
- scripts (gcp scripts)

**Review by boubkerbribri — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Pull request merged**